components/openttd/Makefile

```diff
@@ -10,58 +10,87 @@
 
 #
 # Copyright 2014 Josef 'Jeff' Sipek <jeffpc@josefsipek.net>
+# Copyright 2016 Jim Klimov <jimklimov@gmail.com>
 #
 
 include ../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		openttd
-COMPONENT_VERSION=	1.5.1
+COMPONENT_VERSION=	1.5.3
 COMPONENT_PROJECT_URL=	http://www.openttd.org/en/
 COMPONENT_FMRI=		games/openttd
-COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
-COMPONENT_ARCHIVE=	$(COMPONENT_SRC)-source.tar.gz
-COMPONENT_ARCHIVE_HASH=	\
-	sha256:d856e9c3947955ef0eabaa1793d0829c8bb2e43b8f5942fa328473f9e113419d
-COMPONENT_ARCHIVE_URL=	https://binaries.openttd.org/releases/$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
 COMPONENT_LICENSE=	GPLv2
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 COMPONENT_SUMMARY=	OpenTTD is an open source simulation game based upon Transport Tycoon Deluxe
 COMPONENT_CLASSIFICATION="Applications/Games"
 
-CONFIGURE_DEFAULT_DIRS=no
+include $(WS_TOP)/make-rules/prep.mk
+include $(WS_TOP)/make-rules/ips.mk
 
-include ../../make-rules/prep.mk
-include ../../make-rules/configure.mk
-include ../../make-rules/ips.mk
+prep::
```

Comment:
Don't do this if possible. It's much better to have independent components. If they have inter-dependencies, they should be committed in the necessary order.